### PR TITLE
Dedicated improvements

### PR DESCRIPTION
**Pull request by codecat**
Lots of improvements to the getting started guide for dedicated servers:

* Added info about downloading titlepacks.
* Added info about adding maps to a matchsettings file.
* Added info about command line arguments.
* Formatting and language improvements.

### CODE REVIEW

File: 04.dedicated-server/01.getting-started/docs.md [hunk 1]
```diff
@@ -79,24 +103,82 @@ Note: If you run multiple servers on the same host, port numbers are automatical
...
+The minimal command line to start the server is `ManiaPlanetServer /nodaemon /dedicated_cfg=DedicatedCfgFile /game_settings=MatchSettingsFile`.
 
->>>>> Always use the `/nodaemon` option to be able to see the potential errors.
+>>>>> Always use the `/nodaemon` option to be able to see any potential errors.
 
-*[Complete list of ManiaPlanetServer arguments](../references/command-line)*
+Here's a full list of options you can run the server with:
 
-Please note that without options, the server will be executed in the background which make it difficult to see errors, add the `/nodaemon` option to make it execute in the foreground.
+|  Option  |  Description  |
```
Comment (magnetik) [inline 04.dedicated-server/01.getting-started/docs.md]: I would not put all options here as we have a dedicated page. It's a quick start so I would just not mention them here.

**Comment by magnetik** [comment]
Thanks a lot for this :)

File: 04.dedicated-server/01.getting-started/docs.md (hunk 1 above)
Comment (codecat) [reply 04.dedicated-server/01.getting-started/docs.md]: Ah, makes sense, I thought that link gave me a 404 before so I just re-wrote the entire table. Will add the missing options.

**Comment by codecat** [comment]
By the way, I see `/forceip` and `/bindip` in the list, but they didn't work for me when I tested them yesterday.

**Comment by magnetik** [comment]
I'll merge this for now. I've opened #75 to keep track of the problem

**Pull request merged**